microsoft-power-bi/microsoft-power-bi-quiz.md

```diff
@@ -12,7 +12,7 @@
  - [ ] B, C, D
  - [ ] D
 
-#### Q2. How do you create a chart visual in Power Bl Desktop Report view? (Select all that apply.)
+#### Q2. How do you create a chart visual in Power BI Desktop Report view? (Select all that apply.)
```

Comment:
@baker-jr-john can you please explain why should we rename from BI to BL?

Reply:
I changed _from_ Bl (capital B, lower-case l) _to_ BI.

Reply:
Nice one, OK good! Thank you again for your contribution will merge now!